opencv/src/main/java/org/bytedeco/opencv/presets/opencv_core.java

```diff
@@ -277,54 +277,57 @@ public void map(InfoMap infoMap) {
+                     + "/** Calls {@link #Mat(int, Pointer, boolean) Mat(CV_64FC2, points, copyData)}. */ public Mat(Point2d points, boolean copyData) { this(CV_64FC2, points, copyData); }\n"
+                     + "/** Calls {@link #Mat(int, Pointer, boolean) Mat(CV_32SC3, points, copyData)}. */ public Mat(Point3i points, boolean copyData) { this(CV_32SC3, points, copyData); }\n"
+                     + "/** Calls {@link #Mat(int, Pointer, boolean) Mat(CV_32FC3, points, copyData)}. */ public Mat(Point3f points, boolean copyData) { this(CV_32FC3, points, copyData); }\n"
+                     + "/** Calls {@link #Mat(int, Pointer, boolean) Mat(CV_64FC3, points, copyData)}. */ public Mat(Point3d points, boolean copyData) { this(CV_64FC3, points, copyData); }\n"
+                     + "/** Calls {@link #Mat(int, Pointer, boolean) Mat(CV_64FC4, scalar, copyData)}. */ public Mat(Scalar scalar, boolean copyData) { this(CV_64FC4, scalar, copyData); }\n"
+                     + "/** Calls {@link #Mat(int, Pointer, boolean) Mat(CV_32SC4, scalar, copyData)}. */ public Mat(Scalar4i scalar, boolean copyData) { this(CV_32SC4, scalar, copyData); }\n"
+                     + "/** Calls {@link #Mat(Point, boolean) Mat(points, false)}. */    public Mat(Point points)    { this(points, false); }\n"
+                     + "/** Calls {@link #Mat(Point2f, boolean) Mat(points, false)}. */  public Mat(Point2f points)  { this(points, false); }\n"
+                     + "/** Calls {@link #Mat(Point2d, boolean) Mat(points, false)}. */  public Mat(Point2d points)  { this(points, false); }\n"
+                     + "/** Calls {@link #Mat(Point3i, boolean) Mat(points, false)}. */  public Mat(Point3i points)  { this(points, false); }\n"
+                     + "/** Calls {@link #Mat(Point3f, boolean) Mat(points, false)}. */  public Mat(Point3f points)  { this(points, false); }\n"
+                     + "/** Calls {@link #Mat(Point3d, boolean) Mat(points, false)}. */  public Mat(Point3d points)  { this(points, false); }\n"
+                     + "/** Calls {@link #Mat(Scalar, boolean) Mat(scalar, false)}. */   public Mat(Scalar scalar)   { this(scalar, false); }\n"
+                     + "/** Calls {@link #Mat(Scalar4i, boolean) Mat(scalar, false)}. */ public Mat(Scalar4i scalar) { this(scalar, false); }\n"
 
                      + "public Mat(byte ... b) { this(b, false); }\n"
-                     + "public Mat(byte[] b, boolean signed) { this(1, b.length, signed ? CV_8SC1:CV_8UC1); data().put(b); }\n"
-                     + "public Mat(short ... s) { this(s, true); }\n"
-                     + "public Mat(short[] s, boolean signed) { this(1, s.length, signed ? CV_16SC1:CV_16UC1); new ShortPointer(data()).put(s); }\n"
+                     + "public Mat(short ... s) { this(s, false); }\n"
```

Comment:
You changed `true` to `false`, but this is the `signed`parameter, not `copyData`. It used to be `true` by default.

Reply:
Yes, that was an error I believe. It was false in the case of ShortPointer...

Reply:
I mean, it's an error I made when I originally wrote this a few years ago, not with your update.

Reply:
OK for me

Reply:
Ok